assemblies/features/specs/src/main/feature/feature.xml

```diff
@@ -70,19 +70,18 @@
     </feature>
 
     <!-- jackson -->
-    <feature name="jackson" version="${jackson.tesb.version}">
+    <feature name="jackson-base" version="${jackson.tesb.version}">
```

Comment:
Renaming is to avoid collision with the one in camel-karaf ?

Reply:
yes, currently those from camel-karaf are preferred